silverstripe/framework/SS-2015-026-1.yaml

```diff
@@ -4,11 +4,11 @@ cve:       ~
 branches:
     3.0.x:
         time:     2015-11-11 14:31:00
-        versions: [>=3.0.0,<=3.0.14]
+        versions: [>=3.0.0,<3.1.0]
     3.1.x:
         time:     2015-11-11 14:31:00
-        versions: [>3.1.0,<=3.1.16]
+        versions: [>=3.1.0,<3.2.0]
```

Comment:
this is wrong. The silverstripe announcement says that 3.1.16 is fixing the issue too